pandas/tests/arrays/test_boolean.py

```diff
@@ -248,7 +248,11 @@ def test_coerce_to_numpy_array():
     tm.assert_numpy_array_equal(result, expected)
     # with missing values will raise error
     arr = pd.array([True, False, None], dtype="boolean")
-    with pytest.raises(ValueError):
+    msg = str(
+        "cannot convert to 'bool'-dtype NumPy array with missing values."
+        + " Specify an appropriate 'na_value' for this dtype."
```

Comment:
don't need the plus, string literals will be concatenated by the Python parser
```suggestion
        " Specify an appropriate 'na_value' for this dtype."
```

Reply:
Thanks for the suggestion! I have made the changes.